python/tank/authentication/console_authentication.py

```diff
@@ -61,21 +63,29 @@ def authenticate(self, hostname, login, http_proxy):
                 # Insert a \n on the current line so the print is displayed on a new time.
                 print()
                 raise AuthenticationCancelled()
+            except ConsoleLoginWithSSONotSupportedError as e:
+                # SSO login requires a QtApplication environment at this time.
+                print("%s" % e)
+                print()
+                # @TODO: Temporary measure for the Beta program, until we settle
+                #        on a proper way to deal with authentication.
+                #        (c.f. discussion W JF Boismenu regarding Tank and DCCs)
+                sys.exit(-1)
```

Comment:
is this something we should discuss further?

Reply:
Good thing JF is in the office this morning, I just had a chat with him.

I will remove this code, as the exception can simply percolate back up (well, percolation usually work down, but you know what I mean)